integration/dkg/dkg_whiteboard_test.go

```diff
@@ -121,6 +121,13 @@ func createNode(
 	)
 	require.NoError(t, err)
 
+	var hook zerolog.HookFunc = func(e *zerolog.Event, level zerolog.Level, message string) {
```

Comment:
I'm OK with this approach, but can we:
* confirm that this is triggered when expected
* add a comment to where these logs are emitted to note that tests depend on that behaviour

Reply:
Ok, this test confirms that the hook his triggered when expected:
https://github.com/onflow/flow-go/blob/f2d69135e1969d83d2749efdfefd1df4e9b7add2/module/dkg/broker_test.go#L337-L363